include/seqan3/core/simd/view_to_simd.hpp

```diff
@@ -578,10 +586,10 @@ class view_to_simd<urng_t, simd_t>::iterator_type
     std::array<std::ranges::sentinel_t<inner_range_type>, chunk_size> cached_sentinel{};
     //!\brief Pointer to the associated range.
     view_to_simd * this_view{nullptr};
-    //!\brief An array that stores the end position for every sequence within the current chunk.
-    std::array<uint8_t, total_chunks> current_chunk_sizes{};
+    //!\brief Stores the size of all chunks at index 0 and the size of the final chunk at index 1.
+    std::array<uint8_t, 2> current_chunk_sizes{chunk_size, chunk_size};
```

Comment:
as we talked about, this should be made into two variables `chunk_size` and `final_chunk_size`. (since `current_chunk_sizes[0]` stays always the same and is effectively `chunk_size`, we only need `final_chunk_size`.)